Makefile

```
@@ -283,6 +283,38 @@ verify: check format test test-integration test-prometheus
 .PHONY: verify-extended
 verify-extended: check-generate check format test-cov test-cov-clean test-integration test-prometheus
 
+#####################################################################
+# Rules for dev box on GCP                                          #
+#####################################################################
+
+GCP_BOX_TF_DIR  := $(REPO_ROOT)/hack/local-development/gcp
+
+gcp-box-%: export TF_VAR_name = ${USER}-gardener-dev
+gcp-box-%: export TF_VAR_user = ${USER}
```

```suggestion
gcp-box-%: export TF_VAR_name ?= ${USER}-gardener-dev
gcp-box-%: export TF_VAR_user ?= ${USER}
```

In addition, you could transform the username to lower cases to make it more convenient (GCP resources restriction `^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$`)